tools/build_rules/closure/closure_js_binary.bzl

```diff
@@ -46,6 +48,21 @@ _COMPILATION_LEVELS = {
   "advanced": ["--compilation_level=ADVANCED"]
 }
 
+_SUPPORTED_LANGUAGES = {
+  "es3": ["ES3"],
```

Comment:
It's undocumented behavior though. Are the Bazel devs OK with propagating undocumented (or possibly deprecated) behavior to new code?